### DISCUSSION TITLE
Location Information Error

### DISCUSSION THREAD

**Discussion by ghost** [community/community#63346] [GitHub Education]
### Hi everyone,

Hey everybody. I'm a CS student at National University in San Diego, CA. I'm trying to apply for the GitHub Student Developer Pack, but I keep getting the same error that I must modify my browser settings to allow GitHub to use my location information. I have tried using Safari, Chrome and Firefox with all security settings completely turned off but I continue to receive this error. Does anyone have any suggestions? 

![screenshot 2023-08-09 at 8 50 32 PM](https://github.com/community/community/assets/114975653/60fc97cd-b1cc-451c-8037-34f1994e7fc9)

**Comment by nirav-gajera** [comment]
* Try clearing your browser's cache and cookies.
* Try disabling any ad blockers or extensions that you are using.
* Try using a VPN to connect to GitHub.
* Try contacting GitHub Support for help.

**Reply by ghost** [reply]
Thank you for the tips. I tried clearing the browser cache and cookies, disabling any ad blockers and extensions and using a VPN but unfortunately nothing fixed the issue. I think I will be contacting GitHub support for help.

**Reply by nirav-gajera** [reply]
yes , many people's issues have been resolved using this approach. https://support.github.com/request
